requirements.txt

```diff
@@ -186,8 +188,6 @@ social-auth-core==4.1.0
     # via social-auth-app-django
 sqlparse==0.4.2
     # via django
-typing-extensions==4.2.0
```

Comment:
You might actually need this if you run `pip-compile` inside the container with Python 3.7.